{"uid":"72804a22cad04790","name":"Fetching a non-existent book by ID (example #1)","fullName":"Get book by ID.Fetching a non-existent book by ID (example #1)","historyId":"7e4c3196057dd9a9c2888c229c42af69","time":{"start":1736026852197,"stop":1736026852868,"duration":671},"description":"","descriptionHtml":"","status":"broken","statusMessage":"`cy.request()` failed trying to load:\n                \n                http://localhost:7081/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: connect ECONNREFUSED 127.0.0.1:7081\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: OPTIONS\n                URL: http://localhost:7081/\n                \n                -----------------------------------------------------------\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                https://on.cypress.io/request","statusTrace":"CypressError: `cy.request()` failed trying to load:\n                \n                http://localhost:7081/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: connect ECONNREFUSED 127.0.0.1:7081\n                \n                -----------------------------------------------------------\n                \n                The request we sent was:\n                \n                Method: OPTIONS\n                URL: http://localhost:7081/\n                \n                -----------------------------------------------------------\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                https://on.cypress.io/request\n                    at <unknown> (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:132904:70)\n                    at tryCatcher (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1830:23)\n                    at <unknown> (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:4226:37)\n                    at tryCatcher (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1830:23)\n                    at Promise._settlePromiseFromHandler (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1542:31)\n                    at Promise._settlePromise (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1599:18)\n                    at Promise._settlePromise0 (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1644:10)\n                    at Promise._settlePromises (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1720:18)\n                    at _drainQueueStep (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:2430:12)\n                    at _drainQueue (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:2423:9)\n                    at Async._drainQueues (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:2439:5)\n                    at Async.drainQueues (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:2309:14)\n                From Your Spec Code:\n                    at Login.loginUser (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:19115:21)\n                    at sendLoginRequest (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:19275:33)\n                    at Context.eval (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:19289:11)\n                    at Registry.runStepDefininition (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:10419:48)\n                    at Object.fn (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:17018:43)\n                    at runStepWithLogGroup (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:16478:29)\n                    at Context.eval (https://opensource-demo.orangehrmlive.com/__cypress/tests?p=cypress/e2e/tests/api/get/getBookById.feature:17014:62)\n                \n                From Node.js Internals:\n                  RequestError: Error: connect ECONNREFUSED 127.0.0.1:7081\n                      at new r (<embedded>:227:404489)\n                      at ee.callback (<embedded>:227:410490)\n                      at e.callback.s.callback [as _callback] (<embedded>:227:409936)\n                      at s._callback.s.callback.s.callback (<embedded>:290:100275)\n                      at ee.emit (node:events:514:28)\n                      at ee.onRequestError (<embedded>:290:109217)\n                      at ClientRequest.emit (node:events:526:35)\n                      at Socket.socketErrorListener (node:_http_client:501:9)\n                      at Socket.emit (node:events:514:28)\n                      at emitErrorNT (node:internal/streams/destroy:151:8)\n                      at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n                      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"Get book by ID"},{"name":"testClass","value":"Get book by ID"},{"name":"testMethod","value":"Fetching a non-existent book by ID (example #1)"},{"name":"package","value":"Get book by ID"},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"72804a22cad04790.json","parameterValues":[]}